src/interfaces/DragonchainClientInterfaces.ts

```diff
@@ -133,6 +133,18 @@ export interface DragonchainTransactionCreateResponse {
   transaction_id: string
 }
 
+export interface TransactionID {
+  transaction_id: string
+}
+
+export interface DragonchainBulkTransactionCreateResponse {
+  status: number,
+  response: {
+    201: [TransactionID],
+    400: [DragonchainBulkTransactions]
```

Comment:
I wouldn't use the `DragonchainBulkTransactions` here
```suggestion
        400: [DragonchainTransactionCreatePayload]
```